pkg/k8s/errors.go

```diff
@@ -0,0 +1,17 @@
+package k8sclient
+
+import (
+	"errors"
+	"strings"
+)
+
+var ErrAPIServerUnavailable = errors.New("kubernetes API server unavailable")
+
+// isAPIServerUnavailable detects common symptoms of an unreachable API server.
+func isAPIServerUnavailable(err error) bool {
+	errStr := err.Error()
+	return strings.Contains(errStr, "TLS handshake timeout") ||
```

Comment:
thought: wondering if we can at all infer anything from the different errors in terms of what might be the issue?

Reply:
random drive-by ;) you can also use apimachinery to check errors https://github.com/openshift/osde2e-common/blob/7d13c0222fee1854e29e4e15d4ada896a1dfbaf4/pkg/clients/openshift/errors.go#L10

Reply:
Backplane doesn't forward the errors in their original state, changed this PR to just use `The cluster could be down or under heavy load` which is the general backplane string for unreachable api.